.github/workflows/ci.yml

```diff
@@ -43,10 +43,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - name: Install Python 3.10
+      - name: Install Python 3.11
```

Comment:
The `pyapp` and `pyramid-app` cookiecutters now use Python 3.11 by default, so the cookiecutter's CI needs to install 3.11 not 3.10